onebusaway-android/src/main/res/values/donottranslate.xml

```diff
@@ -122,4 +125,28 @@
 
     <!-- General strings -->
     <string name="http_prefix">http://</string>
+
+    <!-- Problem reporting additions -->
+    <string name="ri_append_start">\n\n---------------------\n</string>
+    <string name="ri_append_stop_id">stop_id=%1$s;\n</string>
```

Comment:
Let's change this to `gtfs_stop_id`, so we don't confuse people reviewing the issue when `stop_id` is different from `stop_code`.